lib/linguist/languages.yml

```diff
@@ -1411,9 +1411,14 @@ Groff:
   - .n
   - .rno
   - .roff
-  tm_scope: text.groff
+  - .tmac
+  filenames:
+  - mmn
```

Comment:
[Here for instance](https://github.com/search?utf8=%E2%9C%93&q=filename%3A.mmn&type=Code&ref=searchresults), but they do not seem to be Roff :-)